cirq-core/cirq/study/resolver.py

```diff
@@ -143,9 +143,12 @@ def value_of(
                 product *= self.value_of(factor, recursive)
             return product
         if isinstance(value, sympy.Pow) and len(value.args) == 2:
-            return np.power(
-                self.value_of(value.args[0], recursive), self.value_of(value.args[1], recursive)
-            )
+            base = self.value_of(value.args[0], recursive)
+            exponent = self.value_of(value.args[1], recursive)
+            if isinstance(base, numbers.Number):
+                return np.float_power(base, exponent)
+            else:
```

Comment:
Nit: do we need this else line ?

Reply:
We don't need it, just a matter of taste. I removed it!